authentication/views.py

```diff
@@ -224,39 +237,43 @@ def get_post_response_data(request, token_obj) -> dict:
 
 @api_view(["get"])
 @permission_classes([AllowAny])
-def check_registration_setup(request):
-    logger.info(f"Requested checking registration setup from {request.user}.")
+def checkConfiguration(request):
+    logger.info(f"Requested checking configuration from {request.user}.")
+    page = request.query_params.get("page")
     errors = {}
 
-    # email setup
-    if not settings.DEFAULT_FROM_EMAIL:
-        errors["DEFAULT_FROM_EMAIL"] = "required"
-    if not settings.DEFAULT_EMAIL:
-        errors["DEFAULT_EMAIL"] = "required"
+    if page == FrontendPage.REGISTER.value:
```

Comment:
Considering that the path is defined as
`    path(
        "register",
        RegistrationView.as_view(),
        name="auth_register",
    ),`
    
Instead of the enum, you can use  the `django.urls.reverse`  function.


`assert reverse("auth_register") == "/api/auth/register"` 

So the check can be `page == reverse("auth_register").split("/")[-1]`
So, if for any reason the path changes its name, the reverse function will throw an exception (and we can know this)